src/diet.ts

```diff
@@ -302,7 +307,8 @@ function menu(): MenuItem<Note>[] {
    *  > js Item.all().filter((item) => item.fullness > 0 && item.name.indexOf("lasagna") > 0 && getIngredients(item)["savory dry noodles"]).join(", ")
    */
   const lasagnas = $items`fishy fish lasagna, gnat lasagna, long pork lasagna`;
-  const smallEpics = $items`meteoreo, ice rice`.concat([$item`Tea, Earl Grey, Hot`]);
+  // eslint-disable-next-line libram/verify-constants
```

Comment:
What's all this then

Reply:
I forgot we hadn't pushed that libram change yet